db/migrate/20170402131236_add_visible_field_for_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddVisibleFieldForCustomAttributes < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_attributes, :visible, :boolean, default: true
```

Comment:
Do we need the `default` here as well as the `default_value_for`?  I would have expected just the `default_value_for` setting.

cc @Fryguy